app/controllers/v0/mvi_users_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module V0
+  class MviUsersController < ApplicationController
+    before_action { authorize :mvi, :missing_critical_ids? }
+
+    def submit
+      # Caller must be using proxy add in order to complete Intent to File or Disability Compensation forms
+      form_id = params[:id]
+      if [FormProfiles::VA0996::FORM_ID, VA526ez::FORM_ID].exclude?(form_id)
+        raise Common::Exceptions::Forbidden.new(
```

Comment:
Do you think throwing a 403 for two scenarios: `missing_critical_ids?` policy failure and giving the wrong (form) `id` param is confusing?  The latter gets a pretty error message but still...